doc/source/Resources/Contributing.rst

```diff
@@ -2,11 +2,16 @@
 Contributing
 ============
 
-We welcome all code contributions and hope that this guide facilitate an understanding of the PyAEDT code repository. It is important to note that while the PyAEDT software package is maintained by Ansys and all submissions are reviewed thoroughly before merging, our goal is to foster a community that can support user questions and develop new features to make PyAEDT a powerful tool for all users. As such, we welcome and encourage the submittal of questions and code to this repository.
+Overall guidance on contributing to a PyAnsys library appears in the
+`Contributing <https://dev.docs.pyansys.com/overview/contributing.html>`_ topic
+in the *PyAnsys Developer's Guide*. Ensure that you are thoroughly familiar
+ with it and all `Guidelines and Best Practices <https://dev.docs.pyansys.com/guidelines/index.html>`_.
+ 
+ The following contribution information is specific to PyAEDT.
 
-Cloning the Source Repository
---------------------------------
-You can clone the source repository from PyAEDT GitHub and install the latest version in development mode by running:
+Cloning the PyAEDT Repository
+-----------------------------
+Run this code to clone and install the latest AEDT version in development:
```

Comment:
Development mode is an actual "pip install" mode when installing python libraries cloned.
```suggestion
Run this code to clone and install the latest AEDT version in development mode:
```